docs/operator-manual/security.md

```diff
@@ -238,6 +238,15 @@ can be found in [server/server.go](https://github.com/argoproj/argo-cd/blob/abba
 Argo CD does not log IP addresses of clients requesting API endpoints, since the API server is typically behind a proxy. Instead, it is recommended
 to configure IP addresses logging in the proxy server that sits in front of the API server.
 
+### Standard Application log fields
+
+For logs related to an Application, Argo CD will log the following standard fields :
+
+* *application*: the Application name, without the namespace
+* *app-namespace*: the Application's namespace
+* *app-qualified-name*: the complete Application name in the format `<namespace>/<app>` 
```

Comment:
Maybe we can just drop this field, since it's easily derived from the previous two fields?